### DISCUSSION TITLE
How can I get this working for react-app with typescript?

### DISCUSSION THREAD

**Discussion by sankarkumar23** [SimplifyJobs/webpack-ext-reloader#119] [Q&A]
Its not working for react-app with typescript

ERROR in Entry module not found: Error: Can't resolve './src' in ...
It is expecting index.js in src folder but I have index.tsx

**Comment by rushilsrivastava** [comment]
We use typescript + React internally and it works well. The code is only injected at the build step after webpack has already bundled the code into a `js` file. Can you share your setup?

**Reply by sankarkumar23** [reply]
That was my bad.. This is no longer an issue. Here is the actually error after the webpack.dev.js setup.

![image](https://user-images.githubusercontent.com/12550023/134847204-c6abd518-1cee-4d35-966d-c412a6ff4941.png)

here is my package.config setting

"scripts": {
    "start": "webpack --config webpack.dev.js --mode=development --watch",
    "build": "craco build",
    "test": "react-scripts test",
    "eject": "react-scripts eject"    
  },

do I need to eject the config and make the changes? basically I am new to react and node js.

**Reply by sankarkumar23** [reply]
@rushilsrivastava : Appreciate your help in resolving the above issue.
